app/views/publisher/_publisher.mobile.haml

```diff
@@ -15,17 +15,16 @@
           = image_tag "social-media-logos/#{service.provider}-32x32.png",
             title: service.provider.titleize, class: "service_icon dim",
             id: "#{service.provider}", maxchar: "#{service.class::MAX_CHARACTERS}"
-
-    %select{id: "aspect_ids_", class: "form-control", name: "aspect_ids[]"}
-      %option{value: 'public'}
-        = t('public')
-
-      %option{value: 'all_aspects', selected: true}
-        = t('all_aspects')
-
-      - current_user.aspects.each do |aspect|
-        %option{value: aspect.id}
-          = "· #{aspect.name}"
+      .pull-right
+        %select{id: "aspect_ids_", name: "aspect_ids[]", multiple: "multiple"}
+          %option{value: "public"}
+            = t("public")
+          %option{value: "all_aspects", selected: true}
+            = t("all_aspects")
+          %option(data-role="divider")
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))

Reply:
I haven't figured out how to get this to work with the requested syntax.  It seems that in an option block HAML is only allowing very specific fields.  If someone knows how to fix this I'll implement that. I need this field to create the divider.  Alternatively we could take the divider out if we don't want to ignore the style problem and don't care about the divider.